zenodo/modules/theme/static/scss/profiles.scss

```diff
@@ -0,0 +1,53 @@
+#researcher-contact{
+  .g-recaptcha div {
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  textarea#body {
+    height: 100px !important;
+  }
+
+  #error-recaptcha {
+    display: table;
+    margin: 0 auto;
+  }
+}
+
+#orcid-profile{
+  .loader {
+    margin: auto;
+    margin-top: 20px;
+    border: 8px solid #f3f3f3;
+    border-radius: 50%;
+    border-top: 8px solid #3498db;
+    width: 80px;
+    height: 80px;
+    -webkit-animation: spin 2s linear infinite;
+    animation: spin 2s linear infinite;
+  }
+
+  @keyframes spin {
+      0% { transform: rotate(0deg); }
+      100% { transform: rotate(360deg); }
+  }
+}
+
+#researcher-profile, #orcid-profile{
+  @media screen and (max-width: 768px) {
```

Comment:
If possible, extending some existing Bootstrap SASS class might be more clear, since manually handling CSS media queries might become tricky (and is already handled pretty well on Bootstrap's side)